test/secure.test.js

```diff
@@ -271,6 +271,50 @@ test('should call next() when required claims present and have correct values an
     t.true(next.called, 'next() should be eventually called in the middleware.');
 });
 
+test('should call next() when token has correct issuer and valid audience array claim', async t => {
+    const keyPair = await generateKeyPair('RS256')
+    const authorizationHeader = "Bearer " + await getJwt(keyPair.privateKey,'', { aud: ['myclient', 'api.example.com'] });
+    const req = requestMock(authorizationHeader);
+    const res = responseMock();
+    const next = sinon.spy();
+    const jwkServiceMock = async () => keyPair.publicKey
+
+    const expectedClaims = [
+        { name: 'iss', value: 'test' },
+        { name: 'aud', value: 'api.example.com' }
+    ];
+
+    const secure = secureMiddleware(jwkServiceMock, { claims: expectedClaims });
+
+    await secure(req, res, next);
+
+    t.true(next.called, 'next() should be eventually called in the middleware.');
+});
+
+test('should call next() when token has correct issuer and invalid audience array claim', async t => {
```

Comment:
I would call the test "should return 403 when...

Reply:
Fixed